### DISCUSSION TITLE
Consider using postgres:13-alpine image in docs for reduced bloat

### DISCUSSION THREAD

**Discussion by angryziber** [teslamate-org/teslamate#1569] [Ideas]
**Describe the bug**

Not really a bug, but a suggestion.
Also, both docker-compose.yml files could be in git.

**Expected behavior**

```
postgres              13-alpine           e43172f9204f        10 days ago         160MB
postgres              13                  26c8bcd8b719        2 weeks ago         314MB
```

**Comment by adriankumpf** [comment]
Pull requests are welcome! I'll move this to [Ideas 💡](https://github.com/adriankumpf/teslamate/discussions/categories/ideas).

**Reply by tobiasehlert** [reply]
> Pull requests are welcome! I'll move this to [Ideas 💡](https://github.com/adriankumpf/teslamate/discussions/categories/ideas).

I've created PR #1576 for the `-alpine` suggestion :)

**Comment by tobiasehlert** [comment]
> both docker-compose.yml files could be in git.

@angryziber, why would you suggest the docker-compose.yml files to be inside the repo?
You need to create the docker-compose file and after that run the pull command and everything gets setup. So would be no need for a checked in docker-compose file anyway, since you will customize it to your needs anyhow.

If I'm faulty here, please correct me on that one :)
